pkg/ddevapp/dotddev_assets/traefik/dynamic_config.middleware.yaml.example

```diff
@@ -0,0 +1,37 @@
+#ddev-generated
+# In order to use Traefik Middleware plugins that are defined globally in ~/.ddev/traefik/static_config.*.yaml files you must insert here the dynamic YAML config provided by the plugin page->Install Plugin at https://plugins.traefik.io/plugins
```

Comment:
Nit: Try to use readable line lengths here, not everybody has wrapping turned on.